src/pages/login.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import {
+  MDBContainer,
+  MDBRow,
+  MDBCol,
+  MDBCard,
+  MDBCardBody,
+  MDBInput,
+  MDBBtn,
+  MDBIcon,
+  MDBModalFooter,
+} from 'mdbreact';
+import './login.css';
```

Comment:
try to use scss in place of css
```suggestion
import './login.scss';
```